app/templates/components/ui-table/cell/admin/users/cell-user-actions.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="ui unordered list">
+  {{#unless @record}}
+    <div class="item">
+      <button type="button" class="w-full ui button" {{action @props.actions.sendVerificationMail @extraRecords.email}}>
+        {{t "Resend Verification Email"}}
+      </button>
+    </div>
+  {{/unless}}
+  <div class="item">
+    <button type="button" class="w-full ui button" {{action @props.actions.resetPasswordMail @extraRecords.email}}>
+      {{t "Password Reset Email"}}
+    </button>
```

Comment:
@mariobehling the text `Resend Verification Email` is fine here, or it need to be replaced with mail icon?